engine/src/main/java/org/terasology/rendering/assets/skeletalmesh/SkeletalMeshDataBuilder.java

```diff
@@ -75,7 +76,7 @@ public SkeletalMeshDataBuilder addMesh(Bone bone, MeshBuilder builder) {
     public SkeletalMeshDataBuilder addBox(Bone bone, Vector3f offset, Vector3f size, float u, float v) {
         MeshBuilder meshBuilder = new MeshBuilder();
         meshBuilder.setTextureMapper(textureMapper);
-        meshBuilder.addBox(offset, size, u, v);
+        meshBuilder.addBox(JomlUtil.from(offset), JomlUtil.from(size), u, v);
```

Comment:
I'm not really happy about the utility method name... `JomlUtil.from(...)` converts a JOML vector to a TeraMath vector? The naming suggests the other way around to me 🤔 

(nothing major, though)

Reply:
from one to the other is what I was thinking but you can think the same with regards to, one to the other. you can argue it either way and both are easy to interpret and make sense of.

Reply:
also seems somewhat insignificant since use should be avoided in new code and the class is slated to be removed